database/migrations/2024_11_12_130822_drop_assets_and_related_tables.php

```diff
@@ -34,19 +34,21 @@
 </COPYRIGHT>
 */
 
-namespace AdvisingApp\InventoryManagement\Database\Factories;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
 
-use Illuminate\Database\Eloquent\Factories\Factory;
-
-/**
- * @extends \Illuminate\Database\Eloquent\Factories\Factory<\AdvisingApp\InventoryManagement\Models\AssetLocation>
- */
-class AssetLocationFactory extends Factory
-{
-    public function definition(): array
+return new class () extends Migration {
+    public function up(): void
     {
-        return [
-            'name' => fake()->word(),
-        ];
+        Schema::dropIfExists('asset_check_outs');
+        Schema::dropIfExists('asset_check_ins');
+        Schema::dropIfExists('maintenance_activities');
+        Schema::dropIfExists('maintenance_providers');
+        Schema::dropIfExists('assets');
+        Schema::dropIfExists('asset_locations');
+        Schema::dropIfExists('asset_statuses');
+        Schema::dropIfExists('asset_types');
```

Comment:
This really isn't how this migration should work. This should be changed to the more standard way of dropping tables. A migration per table dropped with a `down` that would restore the table to it's last state.

Reply:
Hi @Orrison 

Yep, I recreated migration files for drop table and remove permissions.

Thank you.